buildConfiguration.xml

```diff
@@ -2,7 +2,7 @@
   <dotnetArchitecture>x64</dotnetArchitecture>
   <nugetVersion>3.5.0-rc-1285</nugetVersion>
   <runtimes>net45,net461,netstandard2.0</runtimes>
-  <assemblyVersion>6.5.2</assemblyVersion>
+  <assemblyVersion>6.6.1</assemblyVersion>
```

Comment:
6.6.1? Not 6.6.0?

Reply:
This version ties to our preview (nightly) releases. We will release from rel/6.6.0 branch.